### PR TITLE
EVG-12488: Group code changes table contents by commit in /commit-queue

### PR DESCRIPTION
**Pull request by SupaJoon**
https://jira.mongodb.org/browse/EVG-12488
relies on https://github.com/evergreen-ci/evergreen/pull/4275
Screen Shot 2020-12-23 at 4 47 08 PM

### CODE REVIEW

**Comment by khelif96** [comment]
Can you merge master in?

**Comment by SupaJoon** [comment]
evergreen retry

File: src/gql/queries/get-commit-queue.ts [hunk 1]
```diff
@@ -1,5 +1,7 @@
 import { gql } from "@apollo/client";
+import { CodeChangesTable } from "gql/fragments/codeChangesTable";
 
+/* eslint-disable */
```
Comment (khelif96) [inline src/gql/queries/get-commit-queue.ts:4]: can you disable the `graphql/template-strings` rule instead of eslint in general

**Review by khelif96 — APPROVED** [review]

File: src/gql/queries/get-commit-queue.ts (hunk 1 above)
Comment (SupaJoon) [reply src/gql/queries/get-commit-queue.ts:4]: I was unable to target `graphql/template-strings` with a disable comment. I'm unsure why and didn't look into it too closely. I think this query file will become more ergonomic once we switch over to `.graphql` files

File: src/components/CodeChangesTable/index.tsx [hunk 2]
```diff
@@ -19,23 +22,10 @@ export const CodeChangesTable: React.FC<{
...
-    </FileDiffTextContainer>
-  );
-};
-
-const rowKey = (record: FileDiff): string =>
-  `code_changes_row_${record.diffLink}`;
+const rowKey = (
+  record: CodeChangesTableFileDiffsFragment,
+  index: number
+): string => `${index}`;
```
[the remark this answers is not in the document]
Comment (SupaJoon) [reply src/components/CodeChangesTable/index.tsx:30]: I think record.fileName will do the trick

**Pull request merged**